tests/ui/rust-2024/safe-outside-extern.rs

```diff
@@ -0,0 +1,9 @@
+//@ revisions: gated ungated
```

Comment:
nit: I think you can remove this

Reply:
Yeah, kinda wanted to keep the tests as in-sync as possible -- just deleted a bunch of lines rather than totally rewriting it.